src/Presentation/Console/Commands/InitCommand.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\Console\Commands;
+
+use Doctrine\Migrations\Tools\Console\Command\MigrateCommand;
+use SprykerSdk\Sdk\Core\Appplication\Dependency\Repository\TaskRepositoryInterface;
+use SprykerSdk\Sdk\Core\Appplication\Dependency\TaskManagerInterface;
+use SprykerSdk\Sdk\Core\Appplication\Dto\ReceiverValue;
+use SprykerSdk\Sdk\Infrastructure\Entity\Setting;
+use SprykerSdk\Sdk\Infrastructure\Repository\SettingRepository;
+use SprykerSdk\Sdk\Infrastructure\Service\CliValueReceiver;
+use SprykerSdk\SdkContracts\Entity\SettingInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class InitCommand extends Command
+{
+    /**
+     * @var string
+     */
+    public const NAME = 'sdk:init:hidden-sdk';
```

Comment:
It is not clear for me why some commands have `const NAME` and the other `static $defaultName`. Is it possible to use only one way of defining a command name?

Reply:
As discussed let's use `const` instead of default command. For now it's ok. I'll create separate refactoring task.